MessengerOnFileForLinux/TerminalFunctionality/src/EndConversation.cpp

```diff
@@ -2,14 +2,14 @@
 #include <ConversationControl.hpp>
 #include <FileHandling.hpp>
 
-EndConversation::EndConversation(std::string command, std::shared_ptr<ChatInformation> chatInfo) :
-    TerminalCommand(command)
-  , chatInfo_(chatInfo)
+EndConversation::EndConversation(std::string command, std::shared_ptr<ChatInformation> chatInfo)
```

Comment:
Nie będę wszędzie pisał, ale tak jak rozmawialiśmy dzisiaj. Tam gdzie można stosować const refke/refke to ją stosujemy